backend/src/test/resources/application-test.yml

```
@@ -158,3 +158,6 @@ security:
     not-found-window: 1h
     block-ttl: 24h
 
+roulette:
+  default-adjustment-weight: 0.7
+
```

YAMLlint (1.38.0)

[error] too many blank lines (1 > 0)

(empty-lines)